src/python/tritonfrontend/CMakeLists.txt

```diff
@@ -153,14 +146,21 @@ if(${TRITON_ENABLE_GPU})
 endif()
 
 if(${TRITON_ENABLE_TRACING})
-  target_include_directories(
+    target_include_directories(
       py-bindings
-      PRIVATE ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
+      PUBLIC ${OPENTELEMETRY_CPP_INCLUDE_DIRS}
+    )
+
+    target_link_libraries(
+      py-bindings
+      PUBLIC
+      ${OPENTELEMETRY_CPP_LIBRARIES}
```

Comment:
Now that these libs are exposed publicly by the `tracing-library` that is already linked to the `py-bindings` target, do we need to do it again here?

Reply:
But how can `main`target don't need handling like this? Isn't `main` and `py-bindings` conceptually the same in terms of compilation and linking?

Reply:
Thank you for the suggestion. The new handling of `tracing-library` requires only linking `py-bindings` to `tracing-library` without including/linking any `opentelemetry` directories or libraries directly to `py-bindings`.